In Java, does the finally block get executed if we insert a return statement inside the try block of a try-catch-finally?

The finally block will always run when the try or catch blocks exit unless either the program exits or the thread in which the block are in is killed.